web/modules/custom/hoeringsportal_audit_log/src/Form/SettingsForm.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Drupal\hoeringsportal_audit_log\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\node\Entity\NodeType;
+use Drupal\Core\Url;
+
+/**
+ * Settings form.
+ */
+final class SettingsForm extends ConfigFormBase {
+  public const SETTINGS = 'hoeringsportal_audit_log.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'hoeringsportal_audit_log_config';
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * @phpstan-return array<mixed, mixed>
+   */
+  protected function getEditableConfigNames(): array {
+    return [
+      self::SETTINGS,
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * @phpstan-param array<mixed, mixed> $form
+   * @phpstan-return array<mixed, mixed>
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    // Form constructor.
+    $form = parent::buildForm($form, $form_state);
+    // Default settings.
+    $config = $this->config(self::SETTINGS);
+
+    $url = Url::fromRoute('os2web_audit.plugin_settings_local_tasks');
+    $form['info'] = [
+      '#markup' => '<p>These configurations handle <i>when</i> to create logs, <a href="' . $url->toString() . '">the <code>os2web_audit</code> configuration can be found here.</a></p>',
+      '#title' => $this->t('Logged pages'),
+      '#tree' => TRUE,
+    ];
+
+    $form['logged_pages'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Logged pages'),
+      '#tree' => TRUE,
+    ];
+
+    $form['logged_pages']['logged_route_names'] = [
+      '#type' => 'textfield',
```

Comment:
You should be able to get a list of all routes and let the user pick the interesting ones rather than forcing her to talk to a programmer. Is it a design choice to use a comma list for the routes?

Reply:
hmm I haven't thought of this. I guess I can just call it a design choice.